pkgs/servers/http/unit/default.nix

```diff
@@ -46,13 +60,19 @@ stdenv.mkDerivation rec {
     ++ optional withDebug   [ "--debug" ];
 
   postConfigure = ''
-    ${optionalString withPython     "./configure python  --module=python    --config=${python}/bin/python-config  --lib-path=${python}/lib"}
-    ${optionalString withPHP72      "./configure php     --module=php72     --config=${php72.dev}/bin/php-config  --lib-path=${php72}/lib"}
-    ${optionalString withPHP73      "./configure php     --module=php73     --config=${php73.dev}/bin/php-config  --lib-path=${php73}/lib"}
-    ${optionalString withPerl       "./configure perl    --module=perl      --perl=${perl}/bin/perl"}
-    ${optionalString withPerldevel  "./configure perl    --module=perl529   --perl=${perldevel}/bin/perl"}
-    ${optionalString withRuby_2_4   "./configure ruby    --module=ruby24    --ruby=${ruby_2_4}/bin/ruby"}
-    ${optionalString withRuby       "./configure ruby    --module=ruby      --ruby=${ruby}/bin/ruby"}
+    ${optionalString withPython27   "./configure python --module=python27 --config=${python27}/bin/python2.7-config   --lib-path=${python27}/lib"}
```

Comment:
Please don't add all the separate Python config lines as this is bound to get outdated. Instead, have it once and make it variable. In case of `python3` packages, always use `python3-config` so we don't have to bother with the `m` version.

Reply:
use this variant?
```
, withPython2 ? false, python27
, withPython3 ? true, python37
...
  postConfigure = ''
    ${optionalString withPython2   "./configure python --module=python2 --config=${python27}/bin/python2.7-config  --lib-path=${python27}/lib"}
    ${optionalString withPython3   "./configure python --module=python3 --config=${python37}/bin/python3.7-config  --lib-path=${python37}/lib"}


```